dna-src/zomes/communities/code/src/communities/mod.rs

```diff
@@ -19,22 +19,42 @@ pub struct Community {
     pub slug: String,
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone, DefaultJson)]
+pub struct CommunityResult {
```

Comment:
I've called these structs that are the entry structs but including the address `[Entry]Result` to distinguish them. However, a Result is already a very specific and different thing in Rust. How wrong/confusing is this through the eyes of a Rust dev?

Reply:
also, is there an inheritance mechanism or something like that for structs so that I could avoid the duplication of

```
pub struct Community {
    pub name: String,
    pub slug: String,
}

pub struct CommunityResult {
    pub address: Address,
    pub name: String,
    pub slug: String
}
```

anything like:

```
CommunityResult extends Community {
    pub address: Address
}
```
?